lib/embulk/input/twitter_ads_analytics.rb

```diff
@@ -262,8 +262,8 @@ def get_access_token
       def request_entities(access_token)
         retries = 0
         begin
-          url = "https://ads-api.twitter.com/11/accounts/#{@account_id}/#{entity_plural(@entity).downcase}"
-          url = "https://ads-api.twitter.com/11/accounts/#{@account_id}" if @entity == "ACCOUNT"
+          url = "https://ads-api.twitter.com/12/accounts/#{@account_id}/#{entity_plural(@entity).downcase}"
```

Comment:
Please define constant for API version and use it.

Reply:
I fixed. [6baa0d4](https://github.com/trocco-io/embulk-input-twitter_ads_analytics/pull/12/commits/6baa0d46e75c8ca496de89ed6880b48643c6f5be)